File: app/src/main/java/com/hqq/example/TestWord.java
package com.hqq.example;

import android.content.Context;

import androidx.annotation.NonNull;
import androidx.work.Worker;
import androidx.work.WorkerParameters;

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.example
 * @Date : 14:11
 * @Email : qiqiang213@gmail.com
 * @Describe :
 */
public class TestWord extends Worker {
    public TestWord(@NonNull Context context, @NonNull WorkerParameters workerParams) {
        super(context, workerParams);
    }

    @NonNull
    @Override
    public Result doWork() {


        return Result.success();
    }
}

File: app/src/main/java/com/hqq/example/TestWord.kt
package com.hqq.example

import android.content.Context
import androidx.work.Worker
import androidx.work.WorkerParameters

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.example
 * @Date : 14:11
 * @Email : qiqiang213@gmail.com
 * @Describe :
 */
class TestWord(context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
    override fun doWork(): Result {
        return Result.success()
    }
}